src/components/index.tsx

```diff
@@ -1,17 +1,42 @@
+import { darkTheme, lightTheme, Provider } from "@adobe/react-spectrum";
 import { GoogleOAuthProvider } from "@react-oauth/google";
+import { App } from "components/App";
+import { Luigi } from "components/Luigi";
+import { Stefano } from "components/Stefano";
 import { AuthenticationProvider } from "providers/AuthenticationProvider";
-import { SignatureProvider } from "providers/SignatureProvider";
-import React from "react";
+import { SignatureProvider, useSignatureContext } from "providers/SignatureProvider";
+import React, { FC } from "react";
 import * as ReactDOM from "react-dom";
+import { BrowserRouter, Navigate, Route, Routes } from "react-router-dom";
 
-import App from "./App/App";
+import { Layout } from "./Layout";
 
-const AppWithProviders = () => {
+const AppRouter: FC = () => {
+    const { isDarkMode } = useSignatureContext();
+
+    return (
+        <Provider theme={isDarkMode ? darkTheme : lightTheme} colorScheme="light" height="100%">
+            <Routes>
+                <Route path="/" element={<Layout />}>
+                    <Route path="/" element={<App />} />
+                    <Route path="luigi" element={<Luigi />} />
+                    <Route path="stefano" element={<Stefano />} />
+                </Route>
+
+                <Route path="*" element={<Navigate to="/" replace />} />
```

Comment:
using the wildcard * to handle all non declared routes, where if its anything that is not defined the target path would be root "/" and the replace reserved word to change the browser URL text as to not keep it with the non-existing URL.